src/api/app.js

```diff
@@ -46,3 +65,45 @@ app.get('/', function (req, res) {
 
 const port = 3000;
 app.listen(port, () => logger.info(`Start service on port ${port}!`));
+
+function removeEmailIfOlderThan(email, dateTimeBoundary) {
```

Comment:
This function represents a predicate not a command, maybe we can find a better suited name for it like `emailIsOlderThan(...)`? Also you may consider to inline this function since it is used only once?